axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryDispatchException.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.axonserver.connector.query;
+
+import io.axoniq.axonserver.grpc.ErrorMessage;
+import org.axonframework.axonserver.connector.AxonServerException;
+import org.axonframework.axonserver.connector.ErrorCode;
+import org.axonframework.commandhandling.distributed.CommandDispatchException;
+import org.axonframework.common.AxonException;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * An AxonServer Exception which is thrown on a Query Dispatching exception.
+ *
+ * @author Steven van Beelen
+ * @since 4.0
+ */
+public class AxonServerQueryDispatchException extends AxonException {
```

Comment:
Would you mind adding JavaDoc to getters.

Reply:
Good point, done.